### ISSUE TITLE
Request for updating the article on freeCodeCamp

### ISSUE THREAD

**Issue by Den0702** [TomerPacific/julOnSaleReact#19]
Update please [this](https://www.freecodecamp.org/news/deploy-a-react-app-to-github-pages/) article to make it consistent with the current version of _react-roter-dom_

https://stackoverflow.com/questions/63124161/attempted-import-error-switch-is-not-exported-from-react-router-dom

**Comment by TomerPacific** [comment]
@Den0702  - What exactly is it you want to update? Have you run into an issue?

**Comment by Den0702** [comment]
Hello @TomerPacific, thank you for the answer. 
I ran into an error after following your article (the App.js file). The error is described in the post  link to which I provided above. I solved my problem, however, I think its important to update this article with the working way of declaring and importing Router And Routes. This is because in the current version of _react-router-dom_ they changed the way of importing and declaring Routes.
